conditions.php

```diff
@@ -106,5 +106,5 @@
     /*
      * 3.7.0
      */
-    "user_organisation_access_level"    => "User organisation access level",
+    "user_organisation_access_level"    => "Toegangsniveau gebruikersorganisatie ",
```

Comment:
I'm not 100% sure about this. I don't know how to rephrase this without some more context.

@bytestream?

Reply:
@ju5t Agreed 👍

Reply:
@ju5t funnily enough I had the same discussion when translating Spanish...

It's in the filter conditions:

![image](https://user-images.githubusercontent.com/1788397/142213757-e480aafa-3e91-449a-9dc4-0ef9876f8576.png)

Reply:
@bytestream I have that a lot. English feels more efficient at times. Translations and UX can be frustrating :)

In Dutch, I would probably make this sentence more explicit, like:

> Toegangsniveau gebruiker binnen organisatie

Not sure if this would fit everywhere UX-wise.

Reply:
@ju5t The phrase is only used in the context shown in the screenshot, so if you think that's better then I'm happy to go with that.